### ISSUE TITLE
Namespace packages cause unexpected errors in `repair`

### ISSUE THREAD

**Issue by soltanmm** [pypa/auditwheel#37]
In a project that `auditwheel show` reports is `manylinux1` compatible and that `auditwheel repair` successfully repairs, adding the `namespace_packages` argument to the `setuptools.setup` call results in `auditwheel show` still reporting that the project is `manylinux1` compatible, _however_, `auditwheel repair` breaks:

```
Traceback (most recent call last):
  File "/usr/local/bin/auditwheel", line 11, in <module>
    sys.exit(main())
  File "/opt/_internal/cpython-3.5.1/lib/python3.5/site-packages/auditwheel/main.py", line 49, in main
    rval = args.func(args, p)
  File "/opt/_internal/cpython-3.5.1/lib/python3.5/site-packages/auditwheel/main_repair.py", line 81, in execute
    update_tags=args.UPDATE_TAGS)
  File "/opt/_internal/cpython-3.5.1/lib/python3.5/site-packages/auditwheel/repair.py", line 68, in repair_wheel
    raise RuntimeError('Is this wheel malformatted? Or a bug?')
RuntimeError: Is this wheel malformatted? Or a bug?
```

This is with auditwheel 1.3.0 within the manylinux1 x64 docker image.

Here is a [minimal testcase](https://github.com/pypa/auditwheel/files/255697/asdf.zip) (n.b. requires cython).

**Comment by rmcgibbo** [comment]
Good catch. Thanks for the test cse too.

**Comment by soltanmm** [comment]
@rmcgibbo Any news? Maybe if given a pointer I could try to fix it?

**Comment by ogrisel** [comment]
@soltanmm I don't think any of us has the time to investigate. Please feel free to have a look and ask questions if you do not understand specific sections of the code.

**Comment by soltanmm** [comment]
@ogrisel I can't think of a reason for this check: https://github.com/pypa/auditwheel/blob/master/auditwheel/repair.py#L67 .

Why is it there to begin with? Is it part of auditwheel's spec that it try to handle bad input? Else I'd be inclined to make a PR simply removing it for the sake of namespace packages.

**Comment by rmcgibbo** [comment]
Yes, I think removing that check would be fine.

On Sun, Jun 5, 2016 at 9:35 PM, Masood Malekghassemi <
notifications@github.com> wrote:

> @ogrisel <https://github.com/ogrisel> I can't think of a reason for this
> check:
> https://github.com/pypa/auditwheel/blob/master/auditwheel/repair.py#L67 .
>
> Why is it there to begin with? Is it part of auditwheel's spec that it try
> to handle bad input? Else I'd be inclined to make a PR simply removing it
> for the sake of namespace packages.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pypa/auditwheel/issues/37#issuecomment-223851817>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AAnI_k5iAvSpTNrAvRhg_a8Djv4Rdsllks5qI3lXgaJpZM4Iabzj>
> .
>



-- 
-Robert

**Comment by ehashman** [comment]
Fixed by #45.
